db/migrate/20240215150131_remove_subscription_entitlements_report.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveSubscriptionEntitlementsReport < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
This should be `def up` since there's no way for Rails to reverse it..

Reply:
I am able to revert it with `rollback`.Think Rails is able to manage with `change`. Did you want the same with an empty down method?

Reply:
I doubt `rollback` recreates your deleted report and template inputs, though?

Reply:
No it doesn't..It acts like an empty down method..

Reply:
Okay. `def up` might make that a bit more clear, but if it doesn't error I'm fine with it either way. :+1: